src/content/docs/logs/ui-data/obfuscation-ui.mdx

```diff
@@ -47,20 +51,30 @@ This log record contains several sensitive data. Ideally, you would like your lo
 }
 ```
 
-### 1. What actions to apply [#example-actions]
+Here is the basic process to obfuscate the sensitive data.
```

Comment:
I accidentally skimmed right past this during my first read-through. From a page layout perspective, the collapsers balance the doc in a pleasing way, but they can give the impression that it's 'optional reading.' It doesn't help its situated in the guts of the page (which I find readers tend to zoom right past.) This info seems pretty important for clarifying the nitty-gritty of this obfuscation process. Definitely keep the collapsers, though. My suggestion:

Is there a way to more obviously call this out? I notice you used an H3 for the "Example" in line 28. Maybe because those code blocks are a big red 'Read this' sign anyway, you can get away with moving your H3 allowance to this part of the section, with verbiage that directs/guides the reader, rather than the small line that you have.

Reply:
great idea, thanks!